src/solve.jl

```diff
@@ -26,6 +26,22 @@ end
             (a[1,1]*a[2,2] - a[1,2]*a[2,1])*b[3]) / d )
 end
 
+for Sa in 2:3  # not needed for Sa = 1;
+    @eval begin
+    @inline function solve(::Size{($Sa,$Sa)}, ::Size{Sb}, a::StaticMatrix{<:Any, <:Any, Ta}, b::StaticMatrix{<:Any, <:Any, Tb}) where {Sb, Ta, Tb}
```

Comment:
Following Base, I think it's better if the code inside `@eval` is indented.

Reply:
Also addressed in the latest version.